assets/terraform/aws/bootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Once we are done bootstrapping, we should disable internet egress with iptables. We've shipped regressions of things that needing internet before, without doing that. If we find exceptions that we want to allow in CI, like say NTP, can enable those port by port.

See: https://github.com/gravitational/anypoint/blob/develop/terraform/onprem.tf#L454